examples/amplitude_modulation.rs

```diff
@@ -1,5 +1,5 @@
 use std::{thread, time};
-use web_audio_api::context::{AsBaseAudioContext, AudioContext};
```

Comment:
Simply renaming `AsBaseAudioContext` to `Context` was not the solution we were aiming for.
Can you make this example compile without needing to import the trait? That is the goal! We want the user facing code to not have imports that are not part of the spec.

In any case, please have a look at the failing CI checks

Reply:
In my opinion, AsBaseAudioContext(Context) is used for trait bounds, it would be difficult to remove the traits.
However, you can change it to an enum type, but you still need to import a type that is not in the spec.

https://github.com/tasogare3710/web-audio-api-rs/blob/a163e229bb8ab1c41dd4cc49972ad823ab911029/examples/worklet.rs#L44
etc.

Reply:
Also, only difference between the Context trait and the spec's BaseAudioContext is the name. If you call the Context trait the BaseAudioContext trait, you can call the BaseAudioContext struct the ConcreteBaseAudioContext struct. But, not Rust-ish.